poi_scraper/utils.py

```diff
@@ -32,21 +45,143 @@ def generate_poi_markdown_table(
     return table_header + table_rows
 
 
-def get_url_from_user(ui: UI) -> str:
+def get_base_url(ui: UI) -> str:
     while True:
-        webpage_url = ui.text_input(
+        base_url = ui.text_input(
             sender="Workflow",
             recipient="User",
-            prompt="I can collect Points of Interest (POI) data from any webpage—just share the link with me!",
+            prompt="Great! Please provide the URL of the website you want to collect Points of Interest (POI) data from. Example: https://www.example.com",
         )
-        if is_valid_url(webpage_url):
+        if is_valid_url(base_url):
             break
         ui.text_message(
             sender="Workflow",
             recipient="User",
             body="The provided URL is not valid. Please enter a valid URL. Example: https://www.example.com",
         )
-    return str(webpage_url)
+
+    return str(base_url)
+
+
+def is_unique_name(name: str, db_path: Path) -> bool:
+    try:
+        statement = "SELECT COUNT(*) FROM workflows WHERE name = ?"
+        with get_connection(db_path) as conn:
+            cursor = conn.cursor()
+            cursor.execute(statement, (name,))
+            return bool(cursor.fetchone()[0] == 0)
+    except sqlite3.OperationalError:
+        return True
+
+
+def get_name_for_workflow(ui: UI, db_path: Path) -> str:
```

Comment:
I think it is better to keep word "workflow"  for FastAgency workflows and here use some other name like scrapping task in order to have clear distinction between the two, and not confuse one with another